### ISSUE TITLE
bug function-whitespace-after?

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#631]
```sass
h1 {
    max-height: #{($line-height) * ($lines-to-show)}em;
}
```
Output
```bash
Expected whitespace after ")" (function-whitespace-after) [stylelint]
```
But if i remove all `(`, `)`.
```sass
h1 {
    max-height: #{$line-height * $lines-to-show}em;
}
```
No errors.

**Comment by jeddy3** [comment]
@evilebottnawi FYI, released in `4.0.0`. So you'll need to do a `npm i stylelint@latest`.

**Comment by evilebottnawi** [comment]
@jeddy3 thanks
